features/roomdetails/impl/src/test/kotlin/io/element/android/features/roomdetails/RoomDetailsPresenterTests.kt

```diff
@@ -473,3 +519,19 @@ fun aMatrixRoom(
     isDirect = isDirect,
     notificationSettingsService = notificationSettingsService
 )
+
+@Composable
+private fun <T> withFakeLifecycleOwner(lifecycleOwner: LifecycleOwner? = null, block: @Composable () -> T): T {
```

Comment:
Maybe move this fun to the `testutils` module?

Reply:
https://github.com/element-hq/element-x-android/pull/2274/commits/39bf8217924f79b5d086b5021b27221a68b8678f